scp/charts/configuration/templates/redis/persistentvolume.yaml

```diff
@@ -0,0 +1,19 @@
+{{- $storageClassName := .Values.redis.storage.name -}}
+{{- $capacity := .Values.redis.storage.capacity -}}
+{{- range (untilStep 0 (int .Values.redis.replicaCount) 1) }}
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: local-pv{{ . }}
+spec:
+  storageClassName: {{ $storageClassName }}
+  capacity:
+    storage: {{ $capacity }}
+  accessModes:
+    - ReadWriteOnce
+  hostPath:
+    path: /storage/data{{ . }}
+
+---
+
+{{- end -}}
```

Comment:
I would leave the storage configuration up to the end user. Creating a persistent volume will require elevated permissions and not everyone will have local storage at `/storage/data`.

If you remove this you won't need the storageclass either.

Reply:
I added `configuration.redis.storage.enabled` so these can be turned off

Reply:
I feel like the right approach here is to leave the `PersistentVolume` and `StorageClass` out of this helm chart and leave that up to the cluster administrator to configure the storage options on the cluster. A lot of storage is provisioned dynamically today in kubernetes even on-premise. 

Is there a reason you think these resources would add value to the chart?

Reply:
I'll move the `PersistentVolume` and `StorageClass` Kubernetes YAML to the README